index.src.html

```diff
@@ -683,74 +684,53 @@ <h3 id="determine-requests-referrer">
   associated with it, as detailed in the following steps, which return
   either <code>no referrer</code> or a URL:
 
-  Note: If Fetch is performing a navigation in response to a link of type
-  <code><a>noreferrer</a></code>, then <var>request</var>'s
```

Comment:
Yeah, I thought it would be nice to put it in the switch, because when I was mentally reading that part of the algorithm, I was like, "what about if it's "`no-referrer`"?" and it took me a while to look up and find this note.